src/prefect/indicators/infrastructure/t1.py

```diff
@@ -0,0 +1,172 @@
+"""QualiCharge prefect indicators: infrastructure.
+
+T1: the number of publicly open points of charge by power level.
+"""
+from datetime import datetime
+from string import Template
+from typing import List
+from uuid import UUID
+
+import numpy as np
+import pandas as pd  # type: ignore
+from prefect import flow, runtime, task
+from prefect.artifacts import create_markdown_artifact
+from prefect.futures import wait
+from prefect.task_runners import ThreadPoolTaskRunner
+from sqlalchemy import create_engine
+from sqlalchemy.engine import Connection, Engine
+
+from ..conf import settings
+from ..models import Indicator, IndicatorPeriod, Level
+from ..util import POWER_RANGE, params_level, table_level
+
+VALUES_FOR_TARGETS_TEMPLATE = """
+        WITH
+            $power_range
+        SELECT
+            COUNT(DISTINCT PointDeCharge.id_pdc_itinerance) AS value,
+            category,
+            $level_id AS level_id
+        FROM
+            PointDeCharge
+            INNER JOIN Station ON PointDeCharge.station_id = Station.id
+            INNER JOIN Localisation ON Station.localisation_id = Localisation.id
+            INNER JOIN City ON Localisation.code_insee_commune = City.code
+            LEFT JOIN puissance ON puissance_nominale::numeric <@ category
```

Comment:
`puissance_nominale` casting is required?

Reply:
without cast, i have this error : 
```
ERROR: operator does not exist: double precision <@ numrange 
Hint: No operator matches the given name and argument types. 
You might need to add explicit type casts.
```

Reply:
In postgreSQL doc, there is only one type available for numeric values (numrange). -> i propose to maintain the cast.